### DISCUSSION TITLE
Typo in API documentation: sionna.signal.ifft

### DISCUSSION THREAD

**Discussion by jasperbrg** [NVlabs/sionna#601] [General]
I just noticed a small mistake in the documentation of the IFFT function in the signal submodule. The normalization factor in the documentation is $1/\sqrt{N}$ but actually $\sqrt{N}$ is applied.
Might lead to some confusions :)

Also, I want to suggest adding a power normalization option to the OFDM transmitter utilities somewhere, such that null subcarriers are taken into account in case unit average transmit power is desired. 

Thanks everyone for the great work on Sionna!

**Comment by merlinND** [comment]
Corresponding Issue for the typo you reported: https://github.com/NVlabs/sionna/issues/607
